src/core/functions/aggregate/llm_min.cpp

```diff
@@ -0,0 +1,20 @@
+#include <inja/inja.hpp>
+#include <nlohmann/json.hpp>
+#include <flockmtl/common.hpp>
+#include <flockmtl/core/functions/aggregate.hpp>
+#include <flockmtl/core/functions/aggregate/llm_min_or_max.hpp>
+
+namespace flockmtl {
+namespace core {
+
+void CoreAggregateFunctions::RegisterLlmMinFunction(DatabaseInstance &db) {
+    auto string_concat = AggregateFunction(
+        "llm_min", {LogicalType::VARCHAR, LogicalType::VARCHAR, LogicalType::ANY}, LogicalType::JSON(),
+        AggregateFunction::StateSize<LlmMinOrMaxState>, LlmMinOrMaxOperation::Initialize, LlmMinOrMaxOperation::Operation,
+        LlmMinOrMaxOperation::Combine, LlmMinOrMaxOperation::Finalize<0>, LlmMinOrMaxOperation::SimpleUpdate);
```

Comment:
LlmMinOrMaxOperation::Finalize<0> should change the "0" to something more readable -> "least" I think. Make sure to add the assert to document the possible values.